network-api/networkapi/templates/pages/buyersguide/editorial_content_index_page.html

```diff
@@ -4,75 +4,79 @@
+                        </div>
+                      {% endif %}
 
-                    {% if related_articles %}
-                      <div class="tw-basis-1/2">
-                        {% trans "Popular" as heading %}
-                        {% include "fragments/buyersguide/related_reading.html" with articles=related_articles heading=heading %}
-                      </div>
-                    {% endif %}
-                  </aside>
-                </li>
-              {% endif %}
-            {% endwith %}
-          </ul>
-        {% endif %}
+                      {% if related_articles %}
+                        <div class="tw-basis-1/2">
+                          {% trans "Popular" as heading %}
+                          {% include "fragments/buyersguide/related_reading.html" with articles=related_articles heading=heading %}
+                        </div>
+                      {% endif %}
+                    </aside>
+                  </li>
+                {% endif %}
+              {% endwith %}
+            </ul>
+          {% endif %}
 
-        {% include "fragments/buyersguide/pni_newsletter_box.html" %}
-      </div>
+          {% include "fragments/buyersguide/pni_newsletter_box.html" %}
+        </div>
 
-      {# Desktop aside #}
-      {% with cta=featured_cta related_articles=page.get_related_articles  %}
-        {% if cta or related_articles %}
-          <aside class="tw-hidden large:tw-flex tw-col-span-4 tw-flex-col tw-gap-6">
-            {% if cta %}
-              {% include "fragments/buyersguide/call_to_action_box.html" with icon=cta.sticker_image heading=cta.title body=cta.content link_text=cta.link_label link_href=cta.get_target_url %}
-            {% endif %}
+        {# Desktop aside #}
+        {% with cta=featured_cta related_articles=page.get_related_articles  %}
+          {% if cta or related_articles %}
+            <aside class="tw-hidden large:tw-flex tw-col-span-4 tw-flex-col tw-gap-6">
+              {% if cta %}
+                {% include "fragments/buyersguide/call_to_action_box.html" with icon=cta.sticker_image heading=cta.title body=cta.content link_text=cta.link_label link_href=cta.get_target_url %}
+              {% endif %}
 
-            {% if related_articles %}
-              {% trans "Popular" as heading %}
-              {% include "fragments/buyersguide/related_reading.html" with articles=related_articles heading=heading %}
-            {% endif %}
-          </aside>
-        {% endif %}
-      {% endwith %}
+              {% if related_articles %}
+                {% trans "Popular" as heading %}
+                {% include "fragments/buyersguide/related_reading.html" with articles=related_articles heading=heading %}
+              {% endif %}
+            </aside>
+          {% endif %}
+        {% endwith %}
+      </div>
     </div>
+    {% include "fragments/buyersguide/shape_background.html" %}
```

Comment:
Same question as above: Why is the background not inherited from the `base.html`?

Reply:
removed